implementations/rs/src/networks.rs

```diff
@@ -0,0 +1,76 @@
+    AvalancheFuji = 43113,
+    PalmMainnet = 11297108109,
+    PalmTestnet = 11297108099,
+    AuroraMainnet = 1313161554,
+    AuroraTestnet = 1313161555,
+}
+
+lazy_static! {
+    static ref ALIASES: HashMap<String, KnownNetwork> = {
+        let mut m = HashMap::new();
+        m.insert("1".to_string(), KnownNetwork::Mainnet);
+        m.insert("mainnet".to_string(), KnownNetwork::Mainnet);
+        m.insert("5".to_string(), KnownNetwork::Goerli);
+        m.insert("goerli".to_string(), KnownNetwork::Goerli);
+        m.insert("11155111".to_string(), KnownNetwork::Sepolia);
+        m.insert("sepolia".to_string(), KnownNetwork::Sepolia);
+        m.insert("56".to_string(), KnownNetwork::BinanceSmartChain);
+        m.insert("binance".to_string(), KnownNetwork::BinanceSmartChain);
+        m.insert("42220".to_string(), KnownNetwork::CeloMainnet);
+        m.insert("celo_mainnet".to_string(), KnownNetwork::CeloMainnet);
+        m.insert("44787".to_string(), KnownNetwork::CeloAlfajores);
+        m.insert("celo_alfajores".to_string(), KnownNetwork::CeloAlfajores);
+        m.insert("43114".to_string(), KnownNetwork::AvalancheMainnet);
+        m.insert(
+            "avalanche_mainnet".to_string(),
+            KnownNetwork::AvalancheMainnet,
+        );
+        m.insert("43113".to_string(), KnownNetwork::AvalancheFuji);
+        m.insert("avalanche_fuji".to_string(), KnownNetwork::AvalancheFuji);
+        m.insert("11297108109".to_string(), KnownNetwork::PalmMainnet);
+        m.insert("palm_mainnet".to_string(), KnownNetwork::PalmMainnet);
+        m.insert("11297108099".to_string(), KnownNetwork::PalmTestnet);
+        m.insert("palm_testnet".to_string(), KnownNetwork::PalmTestnet);
+        m.insert("1313161554".to_string(), KnownNetwork::AuroraMainnet);
+        m.insert("aurora_mainnet".to_string(), KnownNetwork::AuroraMainnet);
+        m.insert("1313161555".to_string(), KnownNetwork::AuroraTestnet);
+        m.insert("aurora_testnet".to_string(), KnownNetwork::AuroraTestnet);
+        m
+    };
+    static ref NETWORK_NAMES: HashMap<KnownNetwork, &'static str> = {
+        let mut m = HashMap::new();
+        m.insert(KnownNetwork::Mainnet, "mainnet");
+        m.insert(KnownNetwork::Goerli, "goerli");
+        m.insert(KnownNetwork::Sepolia, "sepolia");
+        m.insert(KnownNetwork::BinanceSmartChain, "binance");
+        m.insert(KnownNetwork::CeloMainnet, "celo-mainnet");
+        m.insert(KnownNetwork::CeloAlfajores, "celo-alfajores");
+        m.insert(KnownNetwork::AvalancheMainnet, "avalanche-mainnet");
+        m.insert(KnownNetwork::AvalancheFuji, "avalanche-fuji");
+        m.insert(KnownNetwork::PalmMainnet, "palm-mainnet");
+        m.insert(KnownNetwork::PalmTestnet, "palm-testnet");
+        m.insert(KnownNetwork::AuroraMainnet, "aurora-mainnet");
+        m.insert(KnownNetwork::AuroraTestnet, "aurora-testnet");
+        m
+    };
+}
+
+pub fn from_alias(alias: &str) -> Option<KnownNetwork> {
```

Comment:
I'm not sure if we want to handle this here, but we want to make sure we handle the case where a chain ID integer is passed as a string (ex: "1")

Reply:
this handles the case of a string passed (i.e `mainnet`) or a chain id as a string (i.e: `"1"`); i need to create test for the latter tho